bundles/statistics/statsgrid2016/Tile.js

```diff
@@ -129,6 +129,23 @@ function(instance, service) {
             el.addClass('material-selected');
         }
     },
+    /**
+     * @method  @public openExtension opens extension
+     * @param  {String} type  flyout type
+     */
+    openExtension: function(type) {
```

Comment:
If you want to add an "open" function it would be nice to have a "close" function and refactor all code using toggle to use these instead.